usage-examples/editorial-commands.ly

```diff
@@ -0,0 +1,34 @@
+\version "2.19.42"
+
+\include "scholarly/package.ly"
+
+\setOption scholarly.editorial.addition #`(
```

Comment:
> This forces the end user to define the editorial functions, otherwise LilyPond will fail.

Will this not be resolved once we use `getChildOptionWithFallback`?

> The syntax with the necessary quasi- and unquoting is daunting for many users.

I agree, but in quoted syntax, `parenthesize`, for example, can not be expanded into a function later, as far as I know. Quasi/un-quote is a small price to pay, but it does work. If you know another way to use OLL's setOption system without doing that (in a less daunting manner), please let me know (or rather, show me, in either a comment or commit). Although, for LP'ers who are using an alist in the first place, I think _switching to backquote and adding a comma_ isn't really *that* much more to digest.

> This more or less restricts the feature to the usage of "simple" functions that can be conveniently expressed within that alist.

As a matter of fact, I agree. That's a condition of LilyPond/Scheme that can't magically be avoided though. The user *can* use long music functions if they predefine them (3538fb2). But maybe I don't understand. Did you not suggest recently on the devel thread I started to use OLL options? This functionality does that.

> These functions are defined in another submodule of ScholarLY. So they can be used independently from the annotation interface.

Yes, it would be handy to have some predefined functions at our (i.e. the user's) disposal, as well as defaults in general for all (or at least the common) possible editorial types. *Although*, we still want to allow the users to specify their own, such as in the example doc (right?).

I don't have a very strong opinion of what the defaults *should* be for many of these things, so I will defer that to you if you have that in mind.

> - use the apply syntax like it is implemented here
> - "apply" one out of a number of editorial functions (which are more or less abstract for now)

I am not really sure what this means. But if you are suggesting something like `apply = slurDashed`, then why have categories as in `apply = addition / deletion /etc.` at all? But that may not be what you suggest. In any case, I'm guessing this should be brought to the user group for feedback.